plan/optimizer.go

```diff
@@ -32,6 +32,7 @@ const (
 	flagPrunColumns uint64 = 1 << iota
 	flagEliminateProjection
 	flagBuildKeyInfo
+	flagEliminateAgg
 	flagDecorrelate
 	flagMaxMinEliminate
 	flagPredicatePushDown
```

Comment:
shall we rename `flagAggregationOptimize` to `flagAggregationPushDown` now?

Reply:
Oh yes, we should do this. I forgot it.